packages/usdk/packages/upstreet-agent/packages/react-agents/classes/active-agent-object.ts

```diff
@@ -75,6 +76,7 @@ export class ActiveAgentObject extends AgentObject {
     //
 
     this.config = config;
+    this.agentJson = new AgentObject(config);
```

Comment:
`this` is already an `AgentObject` so why do we need to wrap it here?

Reply:
required only the agent's configurable attributes, the config fulfills that purpose so i've removed the agentJson and replaced its usage with "config"